src/App/analytics.js

```diff
@@ -9,43 +9,43 @@ class GA {
   constructor() {
     this.ga = new GoogleAnalytics(isDev ? DEV_TRACK_ID : TRACK_ID);
   }
-  
+
   resetClientId () {
-    this.ga.resetClientId();
+    resetClientId();
   }
 
   async trackScreenview (screenName) {
     const params = {cd: screenName};
     await this.ga.send('screenview',params);
   }
-  
+
   async trackEvent (category, action, label = '', value = 0) {
     const params = {ec: category, ea: action, el: label, ev: value};
     await this.ga.send('event', params);
   }
-  
+
   async trackError (error, fatal = 1) {
     const params = {exd: error, exf:fatal};
     await this.ga.send('exception', params);
   }
 }
 
-class DebugGA {  
-  
+class DebugGA {
+
   resetClientId () {
     console.log('Resetting Google Analytics Client ID');
   }
-  
+
   async trackScreenview (screenName) {
     const params = {cd: screenName};
     console.log('Analytics Screenview: ' + JSON.stringify(params));
   }
-  
+
   async trackEvent (category, action, label = '', value = 0) {
     const params = {ec: category, ea: action, el: label, ev: value};
-    await this.ga.send('event', params);
+    console.log('event', params);
```

Comment:
Why doe this become a `console.log` rather than `send`?

Reply:
This is in the DebugGA class used when we don't want to send hits to google analytics